geopandas_view/view.py

```diff
@@ -214,21 +211,19 @@ def view(
     elif not gdf.crs.equals(4326):
         gdf = gdf.to_crs(4326)
 
-    # Get bounds to specify location and map extent
-    bounds = gdf.total_bounds
-    location = kwargs.pop("location", None)
-    if location is None:
-        x = mean([bounds[0], bounds[2]])
-        y = mean([bounds[1], bounds[3]])
-        location = (y, x)
-
-    # get a subset of kwargs to be passed to folium.Map
-    map_kwds = {i: kwargs[i] for i in kwargs.keys() if i in _MAP_KWARGS}
-    for map_kwd in _MAP_KWARGS:
-        kwargs.pop(map_kwd, None)
-
     # create folium.Map object
     if m is None:
+        # Get bounds to specify location and map extent
```

Comment:
I have moved this bit since we're interested in it only if no `m` is passed. No need to run it otherwise.